src/App/App.tsx

```diff
@@ -2,16 +2,11 @@ import { useState } from 'react';
 import AddItem from 'AddItem/AddItem';
 import TodoList from 'TodoList/TodoList';
 import Options from 'Options/Options';
+import { Element } from 'types';
 
 import s from 'App/index.module.css'
```

Comment:
as CSS-file is in the same directory and **relates to the component** I suppose it is better to use relative path
```suggestion
import s from './index.module.css'
```